src/main/java/module-info.java

```diff
@@ -193,5 +190,9 @@
     requires mslinks;
     requires org.antlr.antlr4.runtime;
     requires org.libreoffice.uno;
+    requires org.junit.jupiter.api;
+    requires org.mockito;
+    requires org.mockito.junit.jupiter;
+    requires javafx.swing;
```

Comment:
no updates here please

DO NOT add test dependencies to the RUNTIME

DO NOT add swing.